docs/api.md

```diff
@@ -0,0 +1,550 @@
+---
+sidebar_position: 1
+sidebar_label: API Docs
+title: API Docs
+description: autobrr is a powerful automation tool. With the help of our API, developers can integrate and extend the functionalities of autobrr into their own applications, tools, or systems.
+keywords: [autobrr, api]
+---
+
+# autobrr API
+
+autobrr is a powerful automation tool. With the help of our API, users and developers can integrate and extend the functionalities of autobrr into their own applications, tools, or systems.
+
+## API Endpoint Reference
+
+The **API Endpoint Reference** provides a comprehensive list of available endpoints for interacting with our API.
+
+**Base URL:** `http://127.0.0.1:7474/api`
+
+### Available Endpoints
+
+| #   | Endpoint Description         | Endpoint Path                       |
+| --- | ---------------------------- | ----------------------------------- |
+| 1   | Liveness Check               | `/healthz/liveness`                 |
+| 2   | Readiness Check              | `/healthz/readiness`                |
+| 3   | Download Clients             | `/download_clients`                 |
+| 4   | Feeds                        | `/feeds`                            |
+| 5   | Specific Feed Status         | `/feeds/<FEED_ID>/enabled`          |
+| 6   | Filters                      | `/filters/`                         |
+| 7   | Specific Filter Status       | `/filters/<FILTER_ID>/enabled`      |
+| 8   | Indexer                      | `/indexer`                          |
+| 9   | Specific Indexer Status      | `/indexer/<INDEXER_ID>/enabled`     |
+| 10  | IRC Networks                 | `/irc`                              |
+| 11  | Restart Specific IRC Network | `/irc/network/<NETWORK_ID>/restart` |
+| 12  | API Keys                     | `/keys`                             |
+| 13  | Notifications                | `/notification`                     |
+| 14  | Release History              | `/release`                          |
+| 15  | Config                       | `/config`                           |
+
+### Authentication
+
+All API requests require an API key for authentication. This key can be generated from your autobrr dashboard. Remember to always keep your API key confidential.
```

Comment:
Might wanna specify where you can create the API key instead of just "your autobrr dashboard"
Users might misunderstand "dashboard" as the autobrr start page and not the settings page..